modules/doc/content/help/development/VSCode.md

```diff
@@ -70,6 +70,9 @@ The following settings are recommended to pass the MOOSE GitHub CI pre-checks:
         "${env.CONDA_PREFIX}/**",
         "/usr/include/**"
     ],
+    "files.associations": {
+        "*.C": "cpp",
```

Comment:
```suggestion
        "*.C": "cpp",
        "*.h", "cpp"
```

I have that in my config. We have snippets for headers as well. Do those work for you without this line?

Reply:
yeah they do

Reply:
adding this removes a zillion other suggestions from the list, which all look useless in our context (bunch of ifdef for all types of things)

let s do it?>

Reply:
I'll leave that up to you. If snippets work that's fine. I have the setting  in my config anyways.

Reply:
updated